### ISSUE TITLE
E2E test util is failing when space is embedded in datastore name

### ISSUE THREAD

**Issue by shuklanirdesh82** [vmware/docker-volume-vsphere#1625]
Steps to repro: create a testbed setup which is having space in the name e.g. `datastore (2)` > run E2E tests

CI Run: https://ci.vmware.run/vmware/docker-volume-vsphere/1281

```
2017/07/22 03:20:45 Attached VM name from Docker CLI is [bash: -c: line 0: syntax error near unexpected token `('
bash: -c: line 0: `docker volume inspect  --format '{{index .Status "attached to VM"}}' BasicTestSuite.TestVolumeLifecycle_volume_884362@datastore1 (2)']
```

/CC @govint @tusharnt

**Comment by govint** [comment]
Suggest please revert the CI datastorr name to have no spaces so that everyone doesn't need to be blocked on this and instead let one person try out e2e tests locally with a datastorr with spaces and fix all the places where the code is broken. There are multiple places where fixes need to be made and no point bringing CI down to have this fixed

**Comment by shaominchen** [comment]
Agree with @govint.

**Comment by ashahi1** [comment]
Fixed as part of PR #1717
